chapter_recurrent-neural-networks/language-models-and-dataset.md

```diff
@@ -225,7 +200,7 @@ def seq_data_iter_random(corpus, batch_size, num_steps):
         batch_indices = example_indices[i:(i+batch_size)]
         X = [data(j) for j in batch_indices]
         Y = [data(j + 1) for j in batch_indices]
-        yield torch.Tensor(X), torch.Tensor(Y)
+        yield d2l.tensor(X), d2l.tensor(Y)
```

Comment:
@mli Same here. Should we really use `d2l.tensor()` instead of just using `torch.tensor()` which is more flexible.